azure-mgmt-consumption/tests/test_mgmt_consumption.py

```diff
@@ -12,6 +12,9 @@
 from devtools_testutils import AzureMgmtTestCase, ResourceGroupPreparer
 
 class MgmtConsumptionTest(AzureMgmtTestCase):
+    
+    reservationOrderId = 'ca69259e-bd4f-45c3-bf28-3f353f9cce9b'
```

Comment:
Please confirm none of these values are confidentials

Reply:
Those reservations belong to the test account that we use